conform-pr/main.go

```diff
@@ -240,13 +241,24 @@ func checkSprint(_ *githubactions.Action, projectFields map[string]graphql.Field
 	return errors.New(msg)
 }
 
-// checkTitle checks if PR's title does not end with dot.
+// checkTitle checks if PR's title does not end with dot
+// and also check if it starts with an imperative verb.
 func checkTitle(_ *githubactions.Action, title string) error {
 	titleRegexp := regexp.MustCompile("[a-zA-Z0-9`'\"]$")
```

Comment:
While we are on it – let's check that it starts with a capital letter (`^[A-Z]`, I guess)

Reply:
Okay sure.